Source/Scene/Cesium3DTileset.js

```diff
@@ -472,6 +479,23 @@ define([
         },
 
         /**
+         * When <code>true</code>, all tiles in view are are loaded.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {Boolean}
+         * @readonly
+         *
+         * @default false
+         */
+        viewComplete : {
```

Comment:
Can we please name this differently since it mixes poorly with the camera view semantics.  Why not stay consistent with the well understood tiles loaded paradigm?